src/rust/ide/src/double_representation.rs

```diff
@@ -5,3 +5,10 @@ pub mod definition;
 pub mod graph;
 pub mod node;
 pub mod text;
+
+// ==============
+// === Consts ===
+// ==============
+
+/// Indentation value defined in language specification.
```

Comment:
What is "indentation defined in language specification" ? Lang spec does not specify indentation as far as I know. What is it used for? Pls write a little more docs about it.

Reply:
The sytnax specification ( https://github.com/luna/enso/blob/master/doc/design/syntax/syntax.md ) states:
> Indentation: Indentation is four spaces, and all tabs are converted to 4 spaces. This is not configurable on purpose

Reply:
I will refer to this doc in the comments

Reply:
```rust
/// Indentation value from language specification:
///
/// Indentation: Indentation is four spaces, and all tabs are converted to 4 spaces. This is not
/// configurable on purpose.
///
/// Link: https://github.com/luna/enso/blob/master/doc/design/syntax/syntax.md#encoding
```